File: Go/Math/aliquotSum.go
package main

func aliquotSum(n int) int {
	var result int = 0

	for i := 1; i < n; i++ {
		if n % i == 0 {
			result += i
		}
	}
	return result
}

func checkUntouchable(n int) bool {
	for i := 1; i <= n*n; i++ {
		if aliquotSum(i) == n {
			return false
		}
	}
	return true
}

func checkAbundant(n int) bool {
	if aliquotSum(n) > n {
		return true
	}
	return false
}

File: JavaScript/Math/aliquotSum.js
const aliquotSum = (n) => {
    let result = 0;

    for(let i = 0; i < n; i++) {
        if(n % i === 0) {
            result += i;
        }
    }
    return result;
}

const checkUntouchable = (n) => {
    for(let i = 0; i < n*n; i++) {
        if(aliquotSum(i) === n) {
            return false;
        }
    }
    return true;
}

const checkAbundant = (n) => {
    if(aliquotSum(n) > n) {
        return true;
    }
    return false;
}

const num = 10;

console.log(aliquotSum(num));